### DISCUSSION TITLE
Firefox - Websocket/Connectivity Issues

### DISCUSSION THREAD

**Discussion by pankgeorg** [fonsp/Pluto.jl#1168] [Questions about using Pluto] [answered]
We've received numerous reports of "heisenbugs" with Firefox; Issues that arise and are hard to reproduce/fix for Firefox users
This is a place to keep information related to various issues that may arise and possible solutions.

Related issues:
#848 Pluto becomes unresponsive in Firefox
#859 Startup is loading forever in Firefox

**Comment by VarLad** [comment, marked as answer]
Note, for people getting 
`error resolving "localhost:1234""/" ("") for "P.Q.R.S": rpc error: code = Unknown desc = no such record`

Thats because of the VPN

**Comment by markNZed** [comment]
I was having this issue with Firefox. Clearing out all the cookies/site data did not help. In the end restarting Firefox fixed the issue. The Pluto site was working, then it stopped working. Eventually I found the issues here and tries with Chrome and it worked, so I restarted Firefox and it started working :(

**Comment by gdalle** [comment]
Hey there! Just checking to see if there is any general solution? I don't have a VPN and my only workaround for now is using Chromium

**Reply by fonsp** [reply]
What is your problem? Can you make a video recording?

**Reply by gdalle** [reply]
Weirdly, it doesn't happen anymore in the latest versions of Julia and Pluto. I will be sure to record it if it ever pops up again!
